packages/jest-config/src/resolveConfigPath.ts

```diff
@@ -111,10 +118,23 @@ const findPackageJson = (pathToResolve: string) => {
   return undefined;
 };
 
-const hasPackageJsonJestKey = (packagePath: string) => {
-  const content = fs.readFileSync(packagePath, 'utf8');
+const resolveJestKey = (packageContent: any, packagePath: string) => {
+  if (hasPackageJsonJestKey(packageContent)) {
+    const {jest} = JSON.parse(packageContent);
+
+    if (jest && typeof jest === 'string') {
+      const resolvedConfigFile = path.resolve(packagePath, jest);
+
+      return isFile(resolvedConfigFile) ? resolvedConfigFile : undefined;
```

Comment:
if the path provided isn't a file, we should throw an error

Reply:
>That because jest key can be a path or a config object

My point is that if you in your config define `"jest": "./some-path"` (i.e. it _is_ a string), and the path it resolves to is _not_ a file, then Jest should error. Not ignore the entry.

Reply:
Nice, guess i understood the point: if path is not a config object and it is an string there is no option to it to not be a file, be a file or throw an error. Working on it for the next push commit

Reply:
yeah, exactly 👍